### DISCUSSION TITLE
Module T How to generate a 1/10 cylinder?

### DISCUSSION THREAD

**Discussion by KetchUUpTT** [neperfepx/neper#461] [Q&A] [answered]
Hi!
I want to know how to use Neper to generate a 1/10 cylinder?

**Comment by rquey** [comment, marked as answer]
What is a 1/10 cylinder, exactly? Please be specific.

The option you might be looking for is [-domain](https://neper.info/doc/neper_t.html#cmdoption-domain).

**Reply by KetchUUpTT** [reply]
Thanks for your answer. One tenth of a cylinder refers to a sector column with only 30 degrees instead of 360 degrees. How can I generate it?

**Reply by rquey** [reply]
Use `-domain planes`. Please see the doc.
